### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by Princeb10**
Portal Employee Table commit by prince bharti

### CODE REVIEW

File: portal-organization-directory/src/main/java/com/portal/organization/directory/repository/EmployeeLevelPositionRepository.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.springframework.stereotype.Repository;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Repository
+public interface EmployeeLevelPositionRepository extends JpaRepository<EmployeeLevelPosition,Integer> {
+
+//
+//    @Query("SELECT e.id AS id, CONCAT(e.firstName,' ',e.lastName) AS empName,ep.position AS position,el.levelName AS empLevel,CONCAT(e2.firstName,' ',e2.lastName) AS manager " +
```
Comment (jdaveOptivalue) [inline portal-organization-directory/src/main/java/com/portal/organization/directory/repository/EmployeeLevelPositionRepository.java:18]: Remove commented code if not required

File: portal-organization-directory/src/main/java/com/portal/organization/directory/service/impl/EmployeeLevelPositionServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    private EmployeeLevelPositionRepository employeeLevelPositionRepository;
+
+    @Override
+    public List<SubordinatesDTO> getSubordinatesByManagerId(Integer managerId) {
+        return employeeLevelPositionRepository.findSubordinatesByManagerId(managerId);
+    }
+
+    @Override
+    public List<OrgDirectoryDTO> getOrganizationDirectory() {
+        return null;
```
Comment (jdaveOptivalue) [inline portal-organization-directory/src/main/java/com/portal/organization/directory/service/impl/EmployeeLevelPositionServiceImpl.java:25]: why returning null here

File: portal-organization-directory/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
+spring.application.name=portal-organization-directory
+spring.datasource.url=jdbc:mysql://localhost:3306/opti_portal_employee1
```
Comment (jdaveOptivalue) [inline portal-organization-directory/src/main/resources/application.properties:2]: assign proper database name

File: portal-organization-directory/src/main/java/com/portal/organization/directory/repository/EmployeeLevelPositionRepository.java (hunk 1 above)
Comment (Princeb10) [reply portal-organization-directory/src/main/java/com/portal/organization/directory/repository/EmployeeLevelPositionRepository.java]: I have removed the commented line and updated the pull request

File: portal-organization-directory/src/main/java/com/portal/organization/directory/service/impl/EmployeeLevelPositionServiceImpl.java (hunk 2 above)
Comment (Princeb10) [reply portal-organization-directory/src/main/java/com/portal/organization/directory/service/impl/EmployeeLevelPositionServiceImpl.java]: I have removed this method, i had used it just for testing purpose

**Pull request merged**